AudreyW_Lab_1.ipynb

```diff
@@ -0,0 +1,598 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "sns.distplot(chix)  \n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 41,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.15729920705028488\n",
+      "0.11384629800665769\n"
+     ]
+    }
+   ],
+   "source": [
+    "sigma_test3 = 1-stats.chi2.cdf(2, df=1)\n",
+    "print(sigma_test3)\n",
+    "\n",
+    "sigma_test4 = 1-stats.chi2.cdf(2.5, df=1)\n",
+    "print(sigma_test4)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 42,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "-1.0056199694085208\n",
+      "-1.2063239772954635\n"
+     ]
+    }
+   ],
+   "source": [
+    "prob_test3 = stats.norm.ppf(sigma_test3)\n",
+    "print(prob_test3)\n",
+    "\n",
+    "prob_test4 = stats.norm.ppf(sigma_test4)\n",
+    "print(prob_test4)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The results from above show one would need to go a very far distance to get to their respective probabilities."
```

Comment:
Maybe make clear that because you are addressing the sigma value now, the distance you refer to is on a normally distributed data set, not the chi-square distribution.